.eslintrc

```diff
@@ -24,7 +24,8 @@
         "packageDir": ["../../", "."]
       }
     ],
-    "jsx-a11y/href-no-hash": "off"
+    "jsx-a11y/href-no-hash": "off",
+    "react/destructuring-assignment": "off"
```

Comment:
I generally like this rule , since it makes it very clear to see whether something comes from `state` or from `props`. However, I understand that there are cases where it may be more elegant to not destructure and just use eg. `this.props.aFunction()`, so disabling it in the top level seems like a good call.